terragrunt/modules/crates-io/compute-static/src/log_line.rs

```diff
@@ -0,0 +1,23 @@
+use std::net::IpAddr;
+
+use derive_builder::Builder;
+use serde::Serialize;
+use time::{Date, Time};
+
+#[derive(Debug, Serialize)]
+#[serde(tag = "version")]
+pub enum LogLine {
+    #[serde(rename = "1")]
+    V1(LogLineV1),
+}
+
+#[derive(Debug, Builder, Serialize)]
+pub struct LogLineV1 {
+    date: Date,
```

Comment:
`date` and `time` are serialized using the `serde-human-readable` feature in the `time` crate. Not sure yet if I really like that. It avoids the need for custom serialization, but it's also a bit implicit.

Reply:
Seems OK for now, but human readable sounds iffy to me. For logs I would expect us to want something less human and more machine readable...

It's not clear to me what this actually ends up being but I think something with ~no ambiguity is better.

Reply:
The default format is the internal integer representation of the time or date, which is impossible to interpret. I've updated the code to write dates using [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339), which should solve for both human- and machine-readability.

```json
{
  "date_time":"2023-02-28T11:17:19.002782375Z"
}
```

Reply:
Sounds great, thanks.